Modules/Loadable/Volumes/Widgets/qSlicerScalarVolumeDisplayWidget.cxx

```diff
@@ -241,7 +241,10 @@ void qSlicerScalarVolumeDisplayWidget::updateHistogram()
   vtkDataArray* voxelValues = pointData ? pointData->GetScalars() : nullptr;
 
   // If there are no voxel values then we completely hide the histogram section
-  d->HistogramGroupBox->setVisible(voxelValues != nullptr);
+  if (d->HistogramGroupBox->testAttribute(Qt::WA_WState_Created))
+  {
+    d->HistogramGroupBox->setVisible(voxelValues != nullptr);
+  }
```

Comment:
@lassoan  Should we remove the second call to `d->Histogram->setDataArray(voxelValues);` introduced in c1ec49d09ffe1d8733bc00272206d3c661dbb03c ?